examples/tutorials/advanced/websockets-example-MNIST/Federated learning with websockets and federated averaging.ipynb

```diff
@@ -319,7 +319,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.5"
+   "version": "3.7.3"
```

Comment:
Q: Also here

Reply:
It just changed when I run that notebook. Because I was using that version. It will run on both of them.

Reply:
Changed back to "3.7.5"